### DISCUSSION TITLE
what is this ipv6 address?

### DISCUSSION THREAD

**Discussion by Leo2442926161** [openthread/openthread#8821] [Q&A] [answered]
Dear All,
A quite simple question, what is this ipv6 address? what's the composition of IID?  thank you.

![image](https://user-images.githubusercontent.com/30258768/222159921-72eb4dc9-67d4-477c-9e35-8599aa0213b7.png)

**Comment by Irving-cl** [comment, marked as answer]
Hi @Leo2442926161,

This address is [Mesh-Local EID](https://openthread.io/guides/thread-primer/ipv6-addressing). The IID is random, chosen after commissioning is complete.

**Reply by Leo2442926161** [reply]
Hi Licao, thank you for your response.
My mistake, I thought  ### fd11:35:0:0:45a:c71d:7ba5:795b/64 is MESH-EID address, what is this one?
I can see OTBR has the same prefix ### fd11:35:0:0

**Reply by jwhui** [reply]
`fd11:35:0:0::/64` looks like a manually configured Unique Local Address (ULA) prefix.

The latest OTBR should automatically generate a ULA (called the Off-Mesh Routable Prefix) - see [Thread Border Router - Bidirectional IPv6 Connectivity and DNS-Based Service Discovery](https://openthread.io/codelabs/openthread-border-router) for more details.

**Reply by Leo2442926161** [reply]
Thank you very much Jonathan,  I got it.
one more question, is Thread device allowed to got a GUA? that I can access to my thread devices everywhere?

**Reply by jwhui** [reply]
Yes, of course. Thread networks may also be configured with a globally-routable IPv6 prefix.

**Reply by Leo2442926161** [reply]
Thanks again
